client/connector/connectoronboarding/retry.go

```diff
@@ -0,0 +1,24 @@
+package connectoronboarding
+
+import (
+	"context"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/connector"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/http"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/retry"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/model/device/status"
+)
+
+func UntilConnectorStatusIsActive(ctx context.Context, client http.Client, readInp connector.ReadByNameInput, readOutp *connector.ReadOutput) retry.Func {
+	return func() (bool, error) {
+		readRes, err := connector.ReadByName(ctx, client, readInp)
+		*readOutp = *readRes
+		if err != nil {
+			return false, err
+		}
+		client.Logger.Printf("connector larStatus: %v\n", readRes.ConnectorStatus)
```

Comment:
Nit: I wouldn't use the term `larStatus` as you've made a bunch of effort to hide `lar` here :)

Reply:
yea, done